doc/ansible/dev/advanced_options.md

```diff
@@ -50,3 +50,7 @@ You may want to manage what your operator watches and the owner references. This
 ```
 ENTRYPOINT ["/usr/local/bin/entrypoint", "--inject-owner-ref=false"]
 ```
+
+If you have created resources without owner reference injection, it is
```

Comment:
I'd probably add a note above (by the other note) that lets them know there is no automatic way to go back once they deploy a CR without ownerReferences